NGitLab/Models/MergeRequestQuery.cs

```diff
@@ -74,6 +74,11 @@ public class MergeRequestQuery
         /// </summary>
         public QueryAssigneeId AssigneeId { get; set; }
 
+        /// <summary>
+        /// Returns the merge requests reviewer ids.
+        /// </summary>
+        public int[] ReviewerIds { get; set; }
```

Comment:
There's no reviewer_ids field for merge request query.

There's a reviewer_id field however